test/Tests.Infrastructure/TestBase.cs

```diff
@@ -382,8 +389,25 @@ public void UseNewLocalServer(IDictionary<string, string> customSettings = null,
 
         public class ServerCreationOptions
         {
+            public class ReservedSocketsToFree
+            {
+                public Socket ReservedSocket { get; set; }
+                public Socket ReservedTcpSocket { get; set; }
+            }
+
             private IDictionary<string, string> _customSettings;
 
+            private ReservedSocketsToFree _reservedSocketsToFree;
```

Comment:
the backing field should have the same name
```suggestion
            private ReservedSocketsToFree _reservedSockets;
```

Reply:
done